client/package.json

```diff
@@ -297,7 +297,7 @@
         "postinstall": "node ./node_modules/vscode/bin/install",
         "pretest": "npm run compile && npm run lint && npm run licchk && npm run updatePackageJSON",
         "test": "node ./node_modules/vscode/bin/test --verbose",
-        "posttest": "node ./node_modules/.bin/remap-istanbul -i ./coverage/coverage.json -o ./coverage/coverage-remap.json && node ./node_modules/.bin/istanbul check-coverage ./coverage/coverage-remap.json",
+        "posttest": "remap-istanbul -i ./coverage/coverage.json -o ./coverage/coverage-remap.json && istanbul check-coverage ./coverage/coverage-remap.json",
```

Comment:
does it still find the command?

Reply:
Yes, npm automatically adds the `.bin` directory into the PATH when it runs any of the scripts.

Reply:
Also, the build works, so of course it does ;-)